### PR TITLE
Fix misspellings

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR fixes a few misspellings.

I used the [misspellings](https://pypi.python.org/pypi/misspellings) tool to locate these:

```
$ find . | misspellings -f -
```

### CODE REVIEW

File: packages/ember/tests/routing/basic_test.js [hunk 1]
```diff
@@ -3697,7 +3697,7 @@ QUnit.test('Doesnt swallow exception thrown from willTransition', function() {
 
   throws(() => {
     run(() => router.handleURL('/other'));
-  }, /boom/, 'expected an exception that didnt happen');
+  }, /boom/, 'expected an exception that didn\'t happen');
```
Comment (locks) [inline packages/ember/tests/routing/basic_test.js]: I _think_  we might want to use backticks here, need confirmation.

**Review by locks — CHANGES_REQUESTED** [review]
Great work! I have a nitpick to confirm.

File: packages/ember/tests/routing/basic_test.js (hunk 1 above)
Comment (olleolleolle) [inline packages/ember/tests/routing/basic_test.js]: Rewording:

```
expected an exception but none was thrown
```

**Review by locks — APPROVED** [review]

**Comment by locks** [comment]
Thank you!

**Pull request merged**